.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,15 +2,16 @@
 Please fill out the information below to expedite the review and merge of your pull request.
 -->
 
-#### What this PR does / why we need it:
-<!-- A clear description of the problem you are trying to solve -->
-
+#### Why we need this PR
+<!--
+Briefly outline the purpose of this PR, whether it's addressing a bug, implementing a new feature, or solving a specific problem.
```

Comment:
I would remove "briefly".... if there is an issue already, yes, briefly is enough. But if not, I would expect a more or less detailed description, depending on the size of the PR...